packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -461,7 +468,14 @@ contract EpochRewards is
    * @return The fraction of floating Gold being used for voting in validator elections.
    */
   function getVotingGoldFraction() public view returns (uint256) {
-    uint256 liquidGold = getCeloToken().totalSupply().sub(getReserve().getReserveGoldBalance());
+    uint256 liquidGold;
+    if (isL2()) {
+      liquidGold = ICeloToken(address(getCeloToken())).allocatedSupply().sub(
```

Comment:
Since we have this calculation at least in two places - should we extract it to standalone method ?

Reply:
changed the `GoldToken.allocatedSupply()` to return the correct value depending on L1 or L2 instead.